### DISCUSSION TITLE
How to add button type to CAccordionButton when used through CAccordionHeader?

### DISCUSSION THREAD

**Discussion by PoeppingT** [coreui/coreui-react#252] [Q&A] [answered]
Hi all,

I'm using the `CAccordion` construct as part of a layered form, very useful! I have just one problem -- because the `CAccordionButton` created by the `CAccordionHeader` component doesn't have `type=button` applied, my Formik form things I'm submitting the form and doing validations before I want. 

Looking at the code, it looks like the `CAccordionButton` could support this if used directly:
```
    return (
      <button
        className={_className}
        {...rest}
        aria-expanded={!visible}
        onClick={() => setVisible(!visible)}
        ref={ref}
      >
        {children}
      </button>
    )
```

but the `CAccordionHeader` does not apply any such type (via the `rest` prop):
```
    return (
      <div className={_className} {...rest} ref={ref}>
        <CAccordionButton>{children}</CAccordionButton>
      </div>
    )
```

Is there a way to pass this type configuration through the `CAccordionHeader` to still make use of the `ref` and collapse functionality?

**Comment by mrholek** [comment, marked as answer]
@PoeppingT it's a bug, I will fix it next release.
